build/ci/pr/Jenkinsfile

```diff
@@ -20,6 +20,12 @@ pipeline {
     }
 
     stages {
+        stage('Build docs') {
```

Comment:
I am not a Jenkins expert but would that not mean that the docs build runs before the tests? I thought we wanted to run it in parallel so that the overall build time does not increase.

Reply:
Thanks for catching this. Fixed